distribution/tools/plugin-cli/build.gradle

```diff
@@ -78,3 +80,28 @@ thirdPartyAudit.ignoreViolations(
   'org.bouncycastle.jcajce.provider.ProvSunTLSKDF$TLSExtendedMasterSecretGenerator',
   'org.bouncycastle.jcajce.provider.ProvSunTLSKDF$TLSExtendedMasterSecretGenerator$2'
 )
+
+thirdPartyAudit.ignoreMissingClasses(
```

Comment:
Why is this needed?

Reply:
Without these, the task `':distribution:tools:plugin-cli:thirdPartyAudit'` fails. See details [here](https://build.ci.opensearch.org/job/gradle-check/14349/consoleFull).

Reply:
> @reta do you know if this is ok?

Yeah, I think this is fine, the common-compress may have optional dependencies on ZSTD and other codecs, so we need to exclude these classes (we don't use these codecs here)